sdk/servicenetworking/arm-servicenetworking/test/public/servicenetworking_operations_test.spec.ts

```diff
@@ -30,7 +30,7 @@ export const testPollingOptions = {
   updateIntervalInMs: isPlaybackMode() ? 0 : undefined,
 };
 
-describe("ServiceNetworking test", () => {
+describe.skip("ServiceNetworking test", () => {
```

Comment:
skip the test cases as 2025-03-01-preview has not been rolled out on portal
![Uploading image.png…]()

FYI @cdxu-147